src/sage/arith/misc.py

```diff
@@ -3498,11 +3497,15 @@ def crt(a, b, m=None, n=None):
 CRT = crt
 
 
-def CRT_list(values, moduli):
+def CRT_list(values, moduli=None):
     r""" Given a list ``values`` of elements and a list of corresponding
     ``moduli``, find a single element that reduces to each element of
     ``values`` modulo the corresponding moduli.
 
+    This function can also be called with one argument, each element
+    of the list is a :class:`IntegerMod_abstract` object. In this case, it returns
+    another :class:`IntegerMod_abstract` object.
```

Comment:
```suggestion
    of the list is a :class:`modular integer <IntegerMod_abstract>`. In this case, it returns
    another :class:`modular integer <IntegerMod_abstract>`.
```
Trying to be a little less technical. Not sure if it is an improvement though.

Reply:
The link should go to https://doc.sagemath.org/html/en/reference/finite_rings/sage/rings/finite_rings/integer_mod.html (I need to fix the link later). The elements aren't called modular integers there either, they're just called elements of ℤ/nℤ.